content/en/ecosystem/distributions.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Distributions
+description: List of OpenTelemetry distributions maintained by third parties.
+---
+
+OpenTelemetry [distributions](/docs/concepts/distributions/) are a way of
+customizing OpenTelemetry components so that they're easier to deploy and use
+with specific observability backends.
+
+Any third-party can customize OpenTelemetry components with back-end, vendor, or
+end-user specific changes. You don't have to use a distributions in order to use
+OpenTelemetry components, though distributions might facilitate usage under
+certain circumstances, such as specific vendor requirements.
+
+The following list contains a sample of OpenTelemetry distributions and the
+component they customize. Note that those are third-party projects and are in
+no way validated or endorsed by the OpenTelemetry community.
```

Comment:
I'd suggestion putting this note inside a `{{% alert title="Note" color="warning" %}}` possibly with revisted prose like this:

> OpenTelemetry **_does not_ validate or endorse** the third-party distributions listed below. The list is provided as a convenience for the community.